### PR TITLE
Fixup for "Enable -Wunused-variable"

### PR DESCRIPTION
**Pull request by Nekotekina**
~~This will probably change few times. For now it almost completely reverts the suspected commit.~~

### CODE REVIEW

File: rpcs3/Emu/Cell/PPUTranslator.cpp [hunk 1]
```diff
@@ -4712,7 +4712,7 @@ void PPUTranslator::SetFPCC(Value* lt, Value* gt, Value* eq, Value* un, bool set
 
 void PPUTranslator::SetFPRF(Value* value, bool set_cr)
 {
-	//const bool is32 =
+	const bool is32 =
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/PPUTranslator.cpp:4715]: for readability I would just write:
```suggestion
	ensure(value->getType()->isFloatTy() || value->getType()->isDoubleTy());
```

File: rpcs3/Input/dualsense_pad_handler.cpp [hunk 2]
```diff
@@ -6,11 +6,11 @@ LOG_CHANNEL(dualsense_log, "DualSense");
 
 namespace
 {
-	//const auto THREAD_SLEEP = 1ms;
-	//const auto THREAD_SLEEP_INACTIVE = 100ms;
+	const auto THREAD_SLEEP = 1ms;
+	const auto THREAD_SLEEP_INACTIVE = 100ms;
```
Comment (Megamouse) [inline rpcs3/Input/dualsense_pad_handler.cpp:10]: i think you can safely remove these two lines as they are copy paste from DS4 (which doesn't use them anyway)

File: rpcs3/Emu/RSX/VK/VKTextureCache.h [hunk 3]
```diff
@@ -495,7 +495,7 @@ namespace vk
 
 		std::unique_ptr<vk::viewable_image> find_temporary_image(VkFormat format, u16 w, u16 h, u16 d, u8 mipmaps)
 		{
-			//const auto current_frame = vk::get_current_frame_id();
+			const auto current_frame = vk::get_current_frame_id();
```
Comment (Megamouse) [inline rpcs3/Emu/RSX/VK/VKTextureCache.h:498]: this should be safe to comment out.

File: rpcs3/Emu/RSX/VK/VKTextureCache.h [hunk 4]
```diff
@@ -511,7 +511,7 @@ namespace vk
 
 		std::unique_ptr<vk::viewable_image> find_temporary_cubemap(VkFormat format, u16 size)
 		{
-			//const auto current_frame = vk::get_current_frame_id();
+			const auto current_frame = vk::get_current_frame_id();
```
Comment (Megamouse) [inline rpcs3/Emu/RSX/VK/VKTextureCache.h:514]: this should be safe to comment out

File: rpcs3/Emu/RSX/Common/ProgramStateCache.cpp [hunk 5]
```diff
@@ -13,7 +13,7 @@ usz vertex_program_utils::get_vertex_program_ucode_hash(const RSXVertexProgram &
 	usz hash = 0xCBF29CE484222325ULL;
 	const void* instbuffer = program.data.data();
 	usz instIndex = 0;
-
+	bool end = false;
```
Comment (Megamouse) [inline rpcs3/Emu/RSX/Common/ProgramStateCache.cpp:16]: I don't see the purpose for this line. So it should be safe to delete.

**Comment by Nekotekina** [comment]
Rebased, I'm looking for regression so this probably needs a retest.

**Comment by Satan86** [comment]
Demon's Souls and Red Dead Redemption now crash again after rebase, didn't before it
logs: 
[des.zip](https://github.com/RPCS3/rpcs3/files/5808238/des.zip)
[rdr.zip](https://github.com/RPCS3/rpcs3/files/5808240/rdr.zip)

**Comment by Nekotekina** [comment]
Rebased again, thanks for testing

**Comment by Satan86** [comment]
Working fine again

**Comment by Nekotekina** [comment]
Anyone to retest?

**Comment by Satan86** [comment]
Now it crashes

**Comment by Nekotekina** [comment]
I think I narrowed the problem and used different workaround for now.

**Comment by Satan86** [comment]
Yeah working again now

File: rpcs3/Emu/Cell/PPUTranslator.cpp (hunk 1 above)
Comment (Nekotekina) [reply rpcs3/Emu/Cell/PPUTranslator.cpp]: Well, could use maybe_unused but seems working like this, just like the rest of the function is commented out.

**Comment by comstock2005** [comment]
This update killed the College Football Revamped game. NCAA Football 14 still works.

**Comment by Nekotekina** [comment]
Please don't write in closed pull requests. Create an issue following the rules, with exact build that introduced a regression.

**Pull request merged**